www/nginx/src/opnsense/scripts/nginx/ngx_functions.js

```diff
@@ -18,9 +20,9 @@ function check_cipher_array(r, browser_ciphers, fingerprint_ciphers, result) {
     browser_cipher = browser_ciphers[browser_cipher_index];
     current_index = fingerprint_ciphers.indexOf(browser_cipher);
     if (current_index === -1 || current_index <= last_index) {
-      // a cipher has been found, which is not supported by the browser
+      // a cipher has been found, which is not supported by the browser or order of preference changed
       // such a connection is definitly intercepted
-      result.status = "Intercepted";
+      result.status = "Intercepted(excess suite or wrong order)";
```

Comment:
since this lands in a header, It may break other's environment. This needs to be announced carefully.
See the other comment